crates/aptos-rosetta-cli/src/construction.rs

```diff
@@ -29,6 +29,35 @@ impl ConstructionCommand {
     }
 }
 
+#[derive(Debug, Parser)]
+pub struct TransactionArgs {
+    #[clap(long)]
+    expiry_offset_secs: Option<i64>,
+    #[clap(long)]
+    sequence_number: Option<u64>,
+    #[clap(long)]
+    max_gas: Option<u64>,
+    #[clap(long)]
+    gas_price: Option<u64>,
+}
+
+impl TransactionArgs {
+    pub fn expiry_time(&self) -> anyhow::Result<u64> {
+        let offset = self.expiry_offset_secs.unwrap_or(60);
+        if offset > 0 {
+            Ok(
+                (SystemTime::now().duration_since(UNIX_EPOCH)?
+                    + Duration::from_secs(offset as u64))
+                .as_secs(),
+            )
+        } else {
+            Ok((SystemTime::now().duration_since(UNIX_EPOCH)?
+                - Duration::from_secs((-offset) as u64))
+            .as_secs())
```

Comment:
why allow setting the expiration time in the past?

Reply:
This is to test the failure case of being too old